packages/devtools_app/lib/src/trace_event.dart

```diff
@@ -104,8 +104,8 @@ class TraceEvent {
     if (_type == null) {
       if (args[typeKey] == 'ui') {
         _type = TimelineEventType.ui;
-      } else if (args[typeKey] == 'gpu') {
-        _type = TimelineEventType.gpu;
+      } else if (args[typeKey] == 'raster') {
```

Comment:
Where are we getting `args[typeKey]` from? Is this data from the VM / engine, and will it have been updated to have the new name? If it is from the engine, perhaps check for both `gpu` and `raster`?

Reply:
We asked the engine team a while back if they could add a UI/GPU distinction bit to the trace args, but they didn't have time to implement this.  So we actually never receive any events with a 'type' arg.  Probably best if we remove this all together.